kernel/src/syscall/chown.rs

```diff
@@ -72,24 +72,24 @@ pub fn sys_fchownat(
         return Ok(SyscallReturn::Return(0));
     }
 
-    let path = {
+    let target = {
```

Comment:
Again, about the naming, I prefer:
```rust
    let path_or_inode = {
        // ...
        fs.lookup_inode(...)
    };
    let inode = path_or_inode.inode();

    if let Some(uid) = uid {
        inode.set_owner(uid)?;
    }
    if let Some(gid) = gid {
        inode.set_group(gid)?;
    }
```

This also applies to several other system calls modified in the same commit.